app/src/main/java/com/example/musicapp/RetrofitClientInstance.kt

```diff
@@ -6,9 +6,8 @@ import retrofit2.converter.gson.GsonConverterFactory
 object RetrofitClientInstance {
 
     private var retrofit: Retrofit? = null;
-
     private val BASE_URL = "https://jsonkeeper.com"
-    val retrofitInstance : Retrofit?
+    val retrofitInstance: Retrofit?
```

Comment:
According to Kotlin best practices, there should only be a space before the colon in the following cases:
-when it's used to separate a type and a supertype;
-when delegating to a superclass constructor or a different constructor of the same class;
-after the object keyword.

Since none of these conditions are met, there should be no space before the colon here. The link to this best practice is https://kotlinlang.org/docs/reference/coding-conventions.html